sim/midas/targetutils/src/main/scala/midas/annotations.scala

```diff
@@ -114,3 +114,40 @@ object ExcludeInstanceAsserts {
       def toFirrtl = ExcludeInstanceAssertsAnnotation(target)
     }
 }
+
+
+//AutoCounter annotations
+
+case class AutoCounterCoverAnnotation(target: ReferenceTarget, label: String, message: String) extends
```

Comment:
Can you make everything you don't want people to use package private

Reply:
It's used by both midas and firesim, so can't really make it package private